host.go

```diff
@@ -499,3 +522,38 @@ func (h *Host) SaveConfig() error {
 	}
 	return nil
 }
+
+func (h *Host) SetInstanceHostnameUbuntu() error {
+	log.Debugf("Setting hostname for: %s", h.Name)
+	cmd, err := h.Driver.GetSSHCommand(fmt.Sprintf(
```

Comment:
I considered this but they are slightly different in other ways (mostly the `/etc/hosts` line) - is it okay to use the command from `SetInstanceHostnameUbuntu` in both styles?